src/gnatss/main.py

```diff
@@ -141,20 +145,29 @@ def clean_tt(
 def get_transmit_times(
     cleaned_travel_times: pd.DataFrame,
     all_gps_solutions: pd.DataFrame,
+    rph_data: Union[pd.DataFrame, None],
```

Comment:
A `Union[pd.DataFrame, None]` essentially makes this an `Optional` input. For better readability, a better practice would be `rph_data: Optional[pd.DataFrame] = None`

Reply:
I believe that `Union` makes more sense here than `Optional`, although they are equivalent. Optional gives the impression that we can choose not to input the `rph_data` argument to the `get_transmit_times` function. Whereas Union gives the impression that the `rph_data` argument can take the value as None. In our case the Optional makes sense. 
I found this FastAPI document that explains this well: [doc](https://fastapi.tiangolo.com/python-types/#using-union-or-optional)
What do you think?

Reply:
I have modified the type to be `rph_data: pd.DataFrame`. This is because the rph_data can never be a None value. It can be an empty dataframe though!